doc/source/reference/random/index.rst

```diff
@@ -202,8 +200,9 @@ Features
    c-api
    Examples of using Numba, Cython, CFFI <extending>
 
-Original Source
-~~~~~~~~~~~~~~~
+Original Source of the Generator and BitGenerators
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 This package was developed independently of NumPy and was integrated in version
 1.17.0. The original repo is at https://github.com/bashtage/randomgen.
```

Comment:
Aren't there more bit generators in `randomgen`? If so, that seems worth mentioning. Otherwise, this subsection is probably best moved to `THANKS.txt`, it doesn't help the user.

Reply:
There are many more there still.